src/components/base/Input/index.js

```diff
@@ -68,6 +68,8 @@ type Props = {
   keepEvent?: boolean,
   onBlur: Function,
   onChange?: Function,
+  onEnter?: Function,
+  onEsc?: Function,
```

Comment:
don't forget `()=>void` is better than Function 🙏  or if it actually take something, put the exact type, if not, we are prone to more blindness on flow types